### PR TITLE
Bug fix in table filter

### PR DESCRIPTION
**Pull request by TBym**
List with a single value gets converted to the value itself, however Pandas isin() function is expecting a list.

### CODE REVIEW

**Comment by Hoxbro** [comment]
Can you add a test in https://github.com/holoviz/panel/blob/main/panel/tests/widgets/test_tables.py. 

Which did not work before this fix and will work after?

**Comment by Hoxbro** [comment]
Can you add a test in https://github.com/holoviz/panel/blob/main/panel/tests/widgets/test_tables.py. 

Which did not work before this fix and will work after?

**Comment by TBym** [comment]
Added function `def test_tabulator_constant_single_element_list_filter_client_side(document, comm)` to test_table.py

**Comment by Hoxbro** [comment]
Your changes breaks panel/tests/widgets/test_tables.py::test_tabulator_constant_list_filter_client_side https://github.com/holoviz/panel/actions/runs/5288423386/jobs/9570931880#step:6:5062

Can you take a look at why this is the case?

**Comment by TBym** [comment]
it should be fine now.

File: panel/widgets/tables.py [hunk 1]
```diff
@@ -483,6 +483,7 @@ def _get_header_filters(self, df):
             elif op == '<=':
                 filters.append(col <= val)
             elif op == 'in':
+                if not isinstance(val, (list, np.ndarray)): val = [val]
```
Comment (Hoxbro) [inline panel/widgets/tables.py:486]: Can you add a comment referencing the issue above this if-statement?

**Review by philippjfr — APPROVED** [review]
Don't want to hold this up further. Very much appreciate the added test!

**Pull request merged**